media-plugins/libvisual-plugins/libvisual-plugins-0.4.0-r4.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools eutils multilib-minimal
+
+PATCHLEVEL=4
+
+DESCRIPTION="collection of visualization plugins for use with the libvisual framework"
+HOMEPAGE="http://libvisual.sourceforge.net/"
+SRC_URI="mirror://sourceforge/libvisual/${P}.tar.gz
+	mirror://gentoo/${P}-patches-${PATCHLEVEL}.tar.bz2
+	mirror://gentoo/${P}-m4-1.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0.4"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x86-fbsd"
+IUSE="alsa debug gtk jack mplayer opengl"
+
+RDEPEND=">=media-libs/fontconfig-2.10.92[${MULTILIB_USEDEP}]
+	~media-libs/libvisual-${PV}[${MULTILIB_USEDEP}]
+	>=x11-libs/libX11-1.6.2[${MULTILIB_USEDEP}]
+	>=x11-libs/libXext-1.3.2[${MULTILIB_USEDEP}]
+	>=x11-libs/libXrender-0.9.8[${MULTILIB_USEDEP}]
+	alsa? ( >=media-libs/alsa-lib-1.0.27.2[${MULTILIB_USEDEP}] )
+	gtk? ( >=x11-libs/gtk+-2.24.23:2[${MULTILIB_USEDEP}] )
+	jack? ( virtual/jack[${MULTILIB_USEDEP}] )
+	opengl? (
+		>=virtual/glu-9.0-r1[${MULTILIB_USEDEP}]
+		>=virtual/opengl-7.0-r1[${MULTILIB_USEDEP}]
+	)"
+DEPEND="${RDEPEND}
+	>=virtual/pkgconfig-0-r1[${MULTILIB_USEDEP}]
+	>=x11-libs/libXt-1.1.4[${MULTILIB_USEDEP}]"
+
+DOCS="AUTHORS ChangeLog NEWS README TODO"
```

Comment:
Please check if removing this line doesn't change the result (as those files are the default in function `einstalldocs`). In case it does change and this is a must line, it should be an array of files.

Reply:
I checked this and removing the line does not change the files being installed